setup-ovs.sh

```diff
@@ -67,7 +75,17 @@ if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
       nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
   else
       nmcli conn mod brcnv-iface connection.autoconnect yes
+      nmcli conn up ovs-slave-$secondary_device
+      # ocp 4.7
+      rm -r /var/ovsbond47 || true
+      cp -r /etc/NetworkManager/system-connections-merged /var/ovsbond47 || true
```

Comment:
if you removed the directory in the former line, this line should fail since the directory is not there. (I see this on coreos-ci)
```suggestion
      mkdir -p /var/ovsbond47 && cp -r /etc/NetworkManager/system-connections-merged /var/ovsbond47 || true
```

Reply:
Strange, this commend is working well. 
If I CP something to a folder that doesn't exists it creates it. 
Its was verified in the tests that i did, but we can test it together again